storage/buckets/getBucketMetadata.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2020 Google LLC
```

Comment:
If we delete the old files, the existing docs that embed the old samples will not work anymore. So, we either need to freeze the versions in those docs or keep the old files around (being sure to avoid duplicated names or issues) until the docs are updated.